selfservice/strategy/oidc/strategy_registration.go

```diff
@@ -90,7 +90,7 @@ func (s *Strategy) processRegistration(w http.ResponseWriter, r *http.Request, a
 
 	vm := jsonnet.MakeVM()
 	vm.ExtCode("claims", jsonClaims.String())
-	evaluated, err := vm.EvaluateSnippet(provider.Config().Mapper, jn.String())
+	evaluated, err := vm.EvaluateAnonymousSnippet(provider.Config().Mapper, jn.String())
 	if err != nil {
```

Comment:
Was marked as an error by golangci-lint (because of go-jsonnet update from 0.16.0 to 0.17.0)